src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -506,9 +559,21 @@
 type: command
 short-summary: Add a custom registry login credential to the task
 examples:
-  - name: Add a registry login credential to a task.
+  - name: Add a registry login credential to a task using an opaque username and password.
```

Comment:
do we wanna use the term "opaque"?

Reply:
Changing this to `plain text` as opaque is for internal use.